libs/cardano-ledger-test/src/Test/Cardano/Ledger/EraClass.hs

```diff
@@ -0,0 +1,155 @@
+-- | Everything you need to build Era polymorphic transactions, all in one place
+module Test.Cardano.Ledger.EraClass (
```

Comment:
Actually, It can't get out of date, because then it won't compile. Perhaps it might be best in the API. It is hard to remember all the names, and where they come from. Only about 60% is in Cardano.Ledger.Core . It is all the exceptions (AonzoEraTx, BabbageEraPParams ...) that are now nicely documented in one place.

Reply:
It's up to you. If it makes it easy for you, feel free to keep it. I am definitely not updating that module.